Source/Diagnostics/WarpXOpenPMD.H

```diff
@@ -176,6 +176,7 @@ private:
     }
   }
 
+  void flushCurrent(bool isBTD) const;
```

Comment:
Would it be possible to add a doc-string to document this function?

Reply:
I added doc for this function

Reply:
Please add the doc in this `.H` file, not the `.cpp` file.

Reply:
done